e2e_test/source_inline/cdc/mysql/mysql_create_drop.slt

```diff
@@ -0,0 +1,218 @@
+# create and drop CDC mysql tables concurrently
+
```

Comment:
This is the new test file.
Differences with before:
- Now the envvars like `MYSQL_USER` are set by `risedev d` (in the `risedev-env` file), no need to manually `export`. BTW, previous hardcoded `-u root` is unnecessary. 
- added shared source test
- also create the users used by risingwave in the slt.